shared/common-adapters/drop-view.native.tsx

```diff
@@ -0,0 +1,32 @@
+import {Platform, requireNativeComponent, View} from 'react-native'
+import * as React from 'react'
+import type * as Styles from '../styles'
+
+const isSupported = Platform.OS === 'ios'
+const IMPL = isSupported ? requireNativeComponent('DropView') : null
+
+export type DropItems = Array<{originalPath?: string; content?: string}>
+export type Props = {
+  children?: React.ReactNode
+  onDropped: (items: DropItems) => void
+  style?: Styles.StylesCrossPlatform
+}
+const DropViewWrapperIOS = (p: Props) => {
```

Comment:
on ios we also allow a per page specific drop flow. only added in chat where it will use the attachment flow instead